resolwe/flow/models/data.py

```diff
@@ -217,7 +217,7 @@ def create_entity(self):
                 entity = entity_query.first()
             else:
 
-                descriptor_schema = DescriptorSchema.objects.get(slug=ds_slug)
+                descriptor_schema = DescriptorSchema.objects.filter(slug=ds_slug).order_by('version').last()
```

Comment:
Can we instead use `.latest()` and define `get_latest_by` meta to use the `version` field by default?